stories/Atom/Lists/lists.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../assets/scss/variables";
+@import "../../assets/scss/breakpoints";
+
+/* unorderlist */
+ul {
+  margin: 0;
+  color: $black;
+  font-size: $font-size-20;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 27.5px;
+}
+
+/* unorderlist */
+ol {
+  margin: 0;
+  color: $black;
+  font-size: $font-size-20;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 27.5px;
+}
+
+/* description list */
+dl {
+  margin: 0;
+  color: $black;
+  font-size: $font-size-20;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 27.5px;
+}
```

Comment:
@Shubham-Sharma-77 ul, ol, dl have same styling so instead of writing multiple times, please combine like ul, ol, dl